docs/index.md

```diff
@@ -1,241 +1,138 @@
----
-hide:
-- navigation
----
-
-# Introducing Pydantic Logfire
-
-From the team behind Pydantic, **Logfire** is an observability platform built on the same belief as our
-open source library — that the most powerful tools can be easy to use.
-
-What sets Logfire apart:
+# Pydantic Logfire
 
-- **Simple and Powerful:** Logfire's dashboard is simple relative to the power it provides, ensuring your entire engineering team will actually use it.
-- **Python-centric Insights:** From rich display of Python objects, to event-loop telemetry, to profiling Python code and database queries, Logfire gives you unparalleled visibility into your Python application's behavior.
-- **SQL:** Query your data using standard SQL — all the control and (for many) nothing new to learn. Using SQL also means you can query your data with existing BI tools and database querying libraries.
-- **OpenTelemetry:** Logfire is an opinionated wrapper around OpenTelemetry, allowing you to leverage existing tooling, infrastructure, and instrumentation for many common Python packages, and enabling support for virtually any language.
-- **Pydantic Integration:** Understand the data flowing through your Pydantic models and get built-in analytics on validations.
+From the team behind Pydantic, Logfire is a new type of observability platform built on the same belief as our open source library — that the most powerful tools can be easy to use.
 
-Pydantic Logfire helps you instrument your applications with less code, less time, and better understanding.
+Logfire is built on OpenTelemetry, and supports monitoring your application from any language, with particularly great support for Python! [Read more](why-logfire/index.md).
 
-## Find the needle in a _stacktrace_
+## Getting Started
 
-![Logfire FastAPI screenshot](images/index/logfire-screenshot-fastapi-200.png)
+This page is a quick walk-through for setting up a Python app:
```

Comment:
Do we have somewhere we can direct people who want to see an example that's not Python?

Reply:
Nope, coming soon